test/system/profile_test.rb

```diff
@@ -103,4 +103,17 @@ def setup
     assert_text I18n.t(:'profiles.personal_access_tokens.index.active_personal_access_tokens',
                        count: @active_token_count - 1)
   end
+
+  test 'can view language selection' do
+    visit profile_preferences_path
+
+    assert_text I18n.t(:'profiles.preferences.locale_form.en')
+  end
```

Comment:
**nitpick:** Can you add aline break between these two tests please.

Reply:
Fixed in 9f4ef54c2cff211eacd6f2db2111499a09aff0d9